tests/modules/nf-core/propr/logratio/main.nf

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PROPR_LOGRATIO } from '../../../../../modules/nf-core/propr/logratio/main.nf'
+
+workflow test_propr_logratio {
```

Comment:
Please implement a test workflow for each of the possible parameterizations, e.g. for testing `alr` etc.